lang/lang-es.js

```diff
@@ -0,0 +1,560 @@
+    userListHeader: 'Usuarios ',
+    commonChatHeader: 'Sala de chat',
+    teacherVideoHeader: 'Vídeo del docente',
+    askQuestionHeader: 'Plantear una duda',
+    questionLable: 'Preguntar',
+    answerLable: 'Responder',
+    commentLable: 'Comentar',
+    markAnswer: 'Marcar como respuesta',
+    markNotes: 'Marcas y notas',
+    askQuestionTimeExceed: 'No se puede editar o eliminar, se ha excedido el límite de tiempo',
+    upvoted: 'No se puede editar o eliminar, se ha agregado un voto positivo',
+    enterText: 'Introduzca el texto antes de publicar',
+    more: '...Más',
+    less: 'Menos',
+    askQuestion: 'Plantear la pregunta',
+    MarksAndNotes: 'Marcas y notas',
+    markAnswerUnmark: 'Primero debe desmarcar la respuesta ya marcada',
+    /* For Chrome */
+    PermissionDeniedErrorExt: '<div class="errorMsg"> Se ha bloqueado el acceso a la cámara. Para proporcionar acceso a la cámara web, siga los siguiente pasos <br />'
+    + 'Ir al ícono de la cámara en la esquina superior derecha de la pantalla <br /> '
+    + 'Haga clic en "Permitir siempre..." y seleccione la opción de cámara <br />  '
+    + 'Acepte y actualice la pantalla  </div> <div class="screenImages">'
+    + ' <figure class="chrome" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-video.png" > <figcaption>Chrome</figcaption> </figure></div>',
+
+    SecurityErrorExt: '<div class="errorMsg"> Se ha bloqueado el acceso a la cámara. Para proporcionar acceso a la cámara web, siga los siguiente pasos <br />'
+    + 'Ir al ícono de la cámara en la esquina superior derecha de la pantalla <br /> '
+    + 'Haga clic en "Permitir siempre..." y seleccione la opción de cámara <br />  '
+    + 'Acepte y actualice la pantalla  </div> <div class="screenImages">'
+    + ' <figure class="chrome" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-video.png" > <figcaption>Chrome</figcaption> </figure></div>',
+
+    nopermissionExt: '<div class="errorMsg"> Se ha bloqueado el acceso a la cámara. Para proporcionar acceso a la cámara web, siga los siguiente pasos <br />'
+    + 'Ir al ícono de la cámara en la esquina superior derecha de la pantalla <br /> '
+    + 'Haga clic en "Permitir siempre..." y seleccione la opción de cámara <br />  '
+    + 'Acepte y actualice la pantalla  </div> <div class="screenImages">'
+    + ' <figure class="chrome" > <img src="https://www.congrea.com/wp-content/uploads/2016/10/ff-video.png" > <figcaption>Chrome</figcaption> </figure></div>',
+
+    chromeExtMiss: "Congrea necesita el plugin'Selector de escritorio' para compartir la pantalla.<br />"
+    + "Puedes descargarlo desde <a href='https://chrome.google.com/webstore/detail/desktop-selector/ijhofagnokdeoghaohcekchijfeffbjl' target='_blank'>AQUÍ.</a>",
+
+
+    /* For Firefox */
+    PermissionDeniedErrorExtFF: '<div class="errorMsg"> Se ha bloqueado el acceso a la cámara. Para proporcionar acceso a la cámara web, siga los siguiente pasos <br />'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 169. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5529949)